### ISSUE TITLE
Unexpected token ' in JSON

### ISSUE THREAD

**Issue by yrojha4ever** [captainsafia/fony#6]
![image](https://cloud.githubusercontent.com/assets/4167541/24786569/543061ac-1b82-11e7-9c8e-d2241cfa4df2.png)

There is issue with recent version of fony while using from command line:
fony -t '{"name": "name"}'

**Comment by shanselman** [comment]
Yes, I'm seeing this on Windows. I've been staring at it, copying from the animated gif, worried I got it wrong:
`
fony -t '{"name": "name", "age": "age", "address": "address"}' -c 2`

**Comment by adlangx** [comment]
Same issue. Using Windows 10 running in ConEmu.

**Comment by shanselman** [comment]
OK, looks like it's Windows' handling of quotes. http://stackoverflow.com/questions/28648473/windows-curl-with-json-data-on-the-command-line 

This works! Just escape the quotes. I'll update the readme.md.

```
C:\github\fony>fony -t "{\"name\": \"name\", \"age\": \"age\", \"address\": \"address\"}" -c 2
[
  {
    "name": "Hannah Campbell",
    "age": 54,
    "address": "498 Lile Point"
  },
  {
    "name": "Jane Gibbs",
    "age": 54,
    "address": "1149 Jezde Boulevard"
  }
]
```

**Comment by captainsafia** [comment]
@shanselman: Thanks for digging into this! I'll add a note about this in the README.

**Comment by shanselman** [comment]
@captainsafia PR for that incoming! No need to bother.

**Comment by adlangx** [comment]
I had to make the outer quotes single tics with the escape chars as well.

Works for me now though. Thanks!

**Comment by shanselman** [comment]
@adlangx This is on Windows using ConEmu? Windows doesn't to single tics, I believe

**Comment by captainsafia** [comment]
Closed in #7.
